GliaWidgets/Public/Glia/Glia+EngagementSetup.swift

```diff
@@ -61,6 +61,11 @@ extension Glia {
         viewFactory: ViewFactory,
         ongoingEngagementMediaStreams: Engagement.Media?
     ) throws {
+        if case let .enqueued(_, enqueudEngagementKind) = interactor.state, enqueudEngagementKind == engagementKind, let rootCoordinator {
```

Comment:
I think comment here will be helpful to understand the reason why we need to early out is such case.
Ideally to add uni test here as well (if possible).